config/samples/dataplane_v1beta1_openstackdataplane.yaml

```diff
@@ -55,5 +117,23 @@ spec:
           edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
           edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
           edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
+
+          gather_facts: false
+          enable_debug: false
+          # edpm firewall, change the allowed CIDR if needed
+          edpm_sshd_configure_firewall: true
+          edpm_sshd_allowed_ranges: ['192.168.122.0/24']
+          # SELinux module
+          edpm_selinux_mode: enforcing
+          edpm_hosts_entries_undercloud_hosts_entries: []
+          # edpm_hosts_entries role
+          edpm_hosts_entries_extra_hosts_entries:
+          - 172.17.0.80 glance-internal.openstack.svc neutron-internal.openstack.svc cinder-internal.openstack.svc nova-internal.openstack.svc placement-internal.openstack.svc keystone-internal.openstack.svc
+          - 172.17.0.85 rabbitmq.openstack.svc
+          - 172.17.0.86 rabbitmq-cell1.openstack.svc
+          edpm_hosts_entries_vip_hosts_entries: []
+          hosts_entries: []
+          hosts_entry: []
+          plan: overcloud
```

Comment:
plan is not needed.

Reply:
done!

Reply:
actually, it was needed: https://github.com/openstack-k8s-operators/edpm-ansible/pull/80